packages/jest-parallel/src/__performance_tests__/workers/jest_parallel.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const pi = require('./pi');
+
+module.exports.loadTest = function() {
+  pi();
```

Comment:
I think if you want to test perf accurately, you need to return the value here back to the parent process.